src/prefect/runner/server.py

```diff
@@ -49,12 +59,61 @@ def _shutdown():
     return _shutdown
 
 
-def start_webserver(
-    runner,
-    log_level: str = None,
-) -> None:
+async def __run_deployment(deployment: "Deployment"):
+    async def _create_flow_run_for_deployment(
+        body: Optional[Dict[Any, Any]] = None
+    ) -> JSONResponse:
+        body = body or {}
+        if deployment.enforce_parameter_schema and deployment.parameter_openapi_schema:
+            try:
+                validate_values_conform_to_schema(
+                    body, deployment.parameter_openapi_schema
+                )
+            except ValueError as exc:
+                raise HTTPException(
+                    status.HTTP_400_BAD_REQUEST,
+                    detail=f"Error creating flow run: {exc}",
+                )
+
+        async with get_client() as client:
+            flow_run = await client.create_flow_run_from_deployment(
```

Comment:
Using `create_flow_run_from_deployment` makes sense due to the existing requirement that a flow run be associated with a deployment when traveling through the flow code resolution mechanics, but it feels like the web server should be able to directly communicate to the runner to start a run for a known deployment.

Can a route dispatch a call to `execute_flow_run` on the `Runner` so the flow run can start sooner? Some changes to `execute_flow_run` or an alternative method might be required, but but I think it's worth exploring.